tests/controllers/controller_with_input_data.py

```diff
@@ -0,0 +1,33 @@
+import enum
+import typing
+
+import dataclasses
+
+import winter
+
+
+class Status(enum.Enum):
+    ACTIVE = 'active'
+    SUPER_ACTIVE = 'super_active'
+
+
+@dataclasses.dataclass
+class InputData:
+    id: int
+    name: str
+    is_god: bool
+    optional_status: typing.Optional[Status]
+    status: Status
+    items: typing.List[int]
+    optional_items: typing.Optional[typing.List[int]]
+    with_default: int = 5
+
+
+@winter.controller
+@winter.route('with-input-data/')
+class ControllerWithInputData:
+
+    @winter.input_data(InputData, to='data')
```

Comment:
I think `input_data` is a too generic term. What do you think about `request_body`?

Reply:
Remove type from the decorator, it's already specified in the type hint. `@winter.input_data('data')` should be enough.